mozetl/clientsdaily/fields.py

```diff
@@ -0,0 +1,177 @@
+from collections import defaultdict
+import pyspark.sql.functions as F
+
+VERSION = 1
+
+
+def mode(l, empty_value='MISSING'):
+    if not l:
+        return empty_value
+    counts = defaultdict(int)
+    for value in l:
+        counts[value] += 1
+    counts = [(v, k) for (k, v) in counts.items()]
+    counts.sort()
+    return counts[-1][1]
+
+
+MAIN_SUMMARY_FIELD_AGGREGATORS = [
```

Comment:
I'd like to keep this reasonably declarative. Do you have a specific suggestion?